cloud/deploy/bundle.go

```diff
@@ -142,11 +142,15 @@ func DeleteBundle(input *DeleteBundleInput) error {
 
 func uploadBundle(tarDirPath, bundlePath, uploadURL string, prependBaseDir bool) (string, error) {
 	tarFilePath := filepath.Join(tarDirPath, "bundle.tar")
+	tarGzFilePath := tarFilePath + ".gz"
 	defer func() {
-		err := os.Remove(tarFilePath)
-		if err != nil {
-			fmt.Println("\nFailed to delete tar file: ", err.Error())
-			fmt.Println("\nPlease delete the tar file manually from path: " + tarFilePath)
+		tarFiles := []string{tarFilePath, tarGzFilePath}
+		for _, file := range tarFiles {
+			err := os.Remove(file)
+			if err != nil {
```

Comment:
nit: I think we should ignore if file does not exists, i.e. if `os.IsNotExist(err)` is true